webapp/src/main/java/com/plhal/ares/webapp/AresApiConfig.java

```diff
@@ -1,38 +1,52 @@
 package com.plhal.ares.webapp;
 
-import com.plhal.ares.model.DataRepository;
-import com.plhal.ares.model.DataRepositoryImpl;
-import com.plhal.ares.model.DataRepositoryProperties;
+import com.plhal.ares.parser.*;
 import com.plhal.ares.service.DataService;
 import com.plhal.ares.service.DataServiceImpl;
+import org.flywaydb.core.Flyway;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.sql.DataSource;
 
 @Configuration
+@Import(DbConfig.class)
 public class AresApiConfig {
 
+    @Bean
+    public Flyway flyway(DataSource dataSource) {
```

Comment:
Toto logicky patri do databazove vrstvy. Mozna vam pomuze poucka, ze ten kod musi zapadnou jen do tech casti, kde je treba (aby ve vyssich vrstach nedoslo k jeho zneuziti). Uplne dole mame datovy modu, Nad tim mame servisni modul a nad tim webovy modul. Pokud pridate do weboveho modulu, tak tim umoznujete dalsim vyvojarum pristupovat primo k databazove vrstve. Nicmene to vy prece nechcete. Pokud byste se rozhodl databazovy modul vyuzit v jine aplikaci jako knihovnu (je to koneckondu jen jar), tak byste mohl mit problem, protoze by vam na webovem modulu nekdo zavedl nejakou zavislost na flyway a to vy prece nechcete. Dalsim argumentem je i to, ze v modulu by melo byt jen nezbytne mnoztvi kodu, aby vyvojar nemusel pro pochopeni dane vrstvy procitat spoustu nesouvysejiciho kodu (clovek, ktery meni UX o datbazove vrstve nemusi nic vedet - maximalne o servisni).